ips.yaml

```diff
@@ -0,0 +1,262 @@
+  - 52.48.244.35
+  - 52.52.34.158
+  - 52.52.95.213
+  - 52.52.118.192
+  - 52.57.132.90
+  - 52.59.46.112
+  - 52.59.147.246
+  - 52.62.12.49
+  - 52.63.142.2
+  - 52.63.164.147
+  - 52.63.167.55
+  - 52.67.148.55
+  - 52.73.209.122
+  - 52.89.43.70
+  - 52.197.31.124
+  - 52.197.224.235
+  - 52.198.25.184
+  - 52.201.3.199
+  - 52.209.34.226
+  - 52.209.186.226
+  - 52.210.232.124
+  - 54.68.48.199
+  - 54.70.202.58
+  - 54.94.206.111
+  - 64.120.6.122
+  - 64.237.49.203
+  - 64.237.55.3
+  - 67.228.213.178
+  - 69.59.28.19
+  - 70.32.40.2
+  - 72.46.130.42
+  - 72.46.140.106
+  - 72.46.153.26
+  - 76.72.167.90
+  - 76.72.167.154
+  - 76.72.172.208
+  - 76.164.194.74
+  - 82.103.136.16
+  - 82.103.139.165
+  - 83.170.113.210
+  - 85.17.156.11
+  - 85.17.156.76
+  - 85.93.93.123
+  - 85.93.93.124
+  - 85.93.93.133
+  - 89.163.146.247
+  - 89.163.242.206
+  - 94.247.174.83
+  - 95.141.32.46
+  - 95.211.198.87
+  - 95.211.217.68
+  - 96.31.66.245
+  - 103.47.211.210
+  - 108.62.115.226
+  - 109.123.101.103
+  - 138.219.43.186
+  - 143.255.57.118
+  - 159.8.146.132
+  - 162.218.67.34
+  - 168.1.92.58
+  - 173.248.147.18
+  - 173.254.206.242
+  - 174.34.156.130
+  - 174.34.162.242
+  - 174.34.224.167
+  - 175.45.132.20
+  - 178.255.152.2
+  - 178.255.153.2
+  - 178.255.154.2
+  - 178.255.155.2
+  - 179.50.12.212
+  - 184.75.208.210
+  - 184.75.209.18
+  - 184.75.210.90
+  - 184.75.210.226
+  - 184.75.214.66
+  - 185.39.146.214
+  - 185.39.146.215
+  - 185.70.76.23
+  - 188.138.40.20
+  - 188.172.252.34
+  - 199.87.228.66
+  - 201.33.21.5
+  - 204.152.200.42
+  - 207.244.80.239
+  - 208.43.68.59
+  - 208.64.28.194
+  - 209.126.117.87
+  - 209.126.120.29
+  - 211.44.63.35
+
+# HMT TrIS domain - core platform of HMT adn its ALBs.
+# Approved by Shaun Seager and Richard Lacey via email 6th June 2018. See commit message
+# for further details.
+hmt_tris:
+  - 62.253.215.99
+  # CIDR ranges
+  - 62.253.215.96/28
+  - 62.253.68.40/29
+  - 62.253.69.56/29
+  - 62.254.17.128/28
+
+# Parole board IP to open access for their Office365 migration.
+parole_board_office_365:
+  - 213.121.161.116
+
+# Office of the Public Guardian
+# Note: DOM1 users route all traffic over VPN so the outbound wifi IPs below are only
+#  relevant for 'OPG digital' team users at OPG.
+opg_digital_team:
+  # Cisco meraki wifi, Axis House, Birmingham.
+  - 213.121.252.154/32
+  # Cisco meraki wifi, Embankment House, Nottingham.
+  - 195.99.201.194/32
+
+laa_south_tyneside:
+  # South Tyneside Vodafone WiFi egress
+  - 85.255.237.0/32
+
+albany_house:
+  # LAA Digital and Security & Privacy, Albany House 9th floor WiFi provided by Nurved
+  - 81.144.174.128/28
+
+#Covering all current digital prisons routing though Cloud Gateway solution.
+# HMP Wayland, HMP Berwyn, HMP Nottingham, HMP Exeter, HMP Liverpool, HMP Kirklevington Grange
+# HMP Holme House, HMP Wandsworth, HMP Highdown, HMP Durham, HMYOI Cookham Wood
+digital_prisons:
+  # egress
+  - 18.130.83.42/32
+  - 52.56.168.163/32
```

Comment:
@Polsonby,  is it file supposed to be stored in this repo?  this file appears as new file, doesn't seem part of upgrade